src/components/navbar/navbar.jsx

```diff
@@ -0,0 +1,24 @@
+import "./navbar.css"
+import { useTheme } from "../../context/themeContext"
+
+export const Navbar =()=>{
+    const { toggleTheme, theme } = useTheme();
```

Comment:
there is a better way to themeing, you can search for adding theme using CSS variables

Reply:
_Thanks a lot. I make sure to implement the same.😊_